android/engine/src/test/java/org/smartregister/fhircore/engine/app/fakes/Faker.kt

```diff
@@ -90,4 +103,202 @@ object Faker {
+      appId = "app",
+      configType = "register",
+      id = "patientRegister",
+      fhirResource =
+        FhirResourceConfig(
+          baseResource = ResourceConfig(resource = "Patient"),
+          relatedResources =
+            listOf(ResourceConfig(resource = "Immunization", searchParameter = "patient"))
+        ),
+      filter =
+        RegisterContentConfig(
+          visible = true,
+          display = "Show overdue",
+          rules = listOf(RuleConfig(name = "filter", condition = "", actions = listOf()))
+        ),
+      searchBar =
+        RegisterContentConfig(
+          visible = true,
+          display = "Search name or ID",
+          computedRules = listOf("patientName")
+        ),
+      registerCard =
+        RegisterCardConfig(
+          rules =
+            listOf(
+              RuleConfig(
+                name = "patientName",
+                condition = "true",
+                actions =
+                  listOf(
+                    "data.put('patientName', fhirPath.extractValue(Patient, 'Patient.name.given') + ' ' + fhirPath.extractValue(Patient, 'Patient.name.family'))"
+                  )
+              ),
+              RuleConfig(
+                name = "patientGender",
+                condition = "true",
+                actions =
+                  listOf(
+                    "data.put('patientGender', fhirPath.extractValue(Patient, 'Patient.gender'))"
+                  )
+              )
+            ),
+          views =
+            listOf(
+              ViewGroupProperties(
+                viewType = ViewType.COLUMN,
+                children =
+                  listOf(
+                    ServiceCardProperties(
+                      viewType = ViewType.SERVICE_CARD,
+                      details =
+                        listOf(
+                          CompoundTextProperties(
+                            viewType = ViewType.COMPOUND_TEXT,
+                            primaryText = "@{patientName}",
+                            primaryTextColor = "#000000"
+                          ),
+                          CompoundTextProperties(
+                            viewType = ViewType.COMPOUND_TEXT,
+                            primaryText = "@{patientGender}",
+                            primaryTextColor = "#5A5A5A"
+                          )
+                        ),
+                      showVerticalDivider = true,
+                      serviceMemberIcons = "PREGNANT_WOMAN,CHILD,CHILD",
+                      serviceButton =
+                        ServiceButton(
+                          visible = true,
+                          text = "1",
+                          status = "OVERDUE",
+                          smallSized = false
+                        )
+                    )
+                  )
+              )
+            )
+        )
+    )
+  }
+
+  fun generateHouseholdRegisterConfiguration(): RegisterConfiguration {
```

Comment:
The assets directory already has these configs, can we read the files instead of hard coding them? Alternatively we can retrieve the configs from the `ConfigurationRegistry` which uses the resources provided in the assets directory.

Reply:
Let me try

Reply:
It works, that's easier than I expected, you code it well